src/Plug.vala

```diff
@@ -59,17 +66,35 @@ namespace Accessibility {
             peripherals_settings =      new Accessibility.Backend.Peripherals ();
             mouse_settings =            new Accessibility.Backend.Mouse ();
             media_keys_settings =       new Accessibility.Backend.MediaKeys ();
-            animations_settings =       new Accessibility.Backend.Animations ();
         }
 
         public override Gtk.Widget get_widget () {
             if (paned == null) {
+
+                var display = new Panes.Display ();
+                var audio = new Panes.Audio ();
+                var typing = new Panes.Typing ();
+                var keyboard = new Panes.Keyboard ();
+                var pointing = new Panes.Pointing ();
+                var clicking = new Panes.Clicking ();
+
+                stack = new Gtk.Stack ();
+                if (animations_settings != null) {
+                    var general = new Panes.General ();
+                    stack.add_titled (general, "general", _("General"));
+                }
+                stack.add_titled (display, "display", _("Display"));
```

Comment:
One thing I'm wondering is if it makes most sense API-wise to use `stack.add_titled` here and use the `title` attribute as in StackSwitcher or to use `stack.add_named` and set `title` in the page itself.

One immediately benefit I can think to using `add_titled` is that it forces the use of `name` which we need to switch pages and makes sure that there is something visibly wrong if you just try to use `stack.add`